t/unit/tasks/test_canvas.py

```diff
@@ -454,6 +455,9 @@ def test_OR(self):
         assert isinstance(ax, _chain)
         assert len(ax.tasks), 3 == 'consolidates chain to chain'
 
+        with pytest.raises(TypeError):
+            assert signature('foo') | None
+
```

Comment:
Please add a subtest that explains this assert.
There's no need to add subtests for all the cases but if we're adding a new assert, we should describe why it is there.

Reply:
Done